cms/envs/common.py

```diff
@@ -1245,7 +1246,7 @@
 
 ############## Settings for Studio Context Sensitive Help ##############
 
-DOC_LINK_BASE_URL = None
+HELP_TOKENS_INI_FILE = REPO_ROOT / "docs" / "cms_config.ini"
```

Comment:
I should have mentioned: there are two pull requests (already merged) to add the new setting:
- https://github.com/edx/edx-internal/pull/53
- https://github.com/edx/configuration/pull/3831
and two more to remove the now-obsolete settings:
- https://github.com/edx/edx-internal/pull/54
- https://github.com/edx/configuration/pull/3832